css/index.css

```diff
@@ -76,18 +90,23 @@ h1 {
     color: white;
     text-decoration: none;
     margin-bottom: 20px;
-     /*======= create a background property. Set the value to #F87B99 */
+    background: #F87B99;
+     /*======= create a background property. Set the value to #F87B99*/
     
   }
   
   .container header nav a:hover {
   /*======= create a background property and set it to white.  Then create a color property and set it's value to pink. */
+  background: white;
+  color pink;
   }
```

Comment:
color pink is incorrect syntax.
color requires a ':'. CSS values are placed after the colon.